ee/tables/jwt/table.go

```diff
@@ -0,0 +1,195 @@
+package kolide_jwt
+
+import (
+	"context"
+	"crypto"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"io"
+	"log/slog"
+	"os"
+	"strings"
+
+	"github.com/golang-jwt/jwt/v5"
+	"github.com/kolide/launcher/ee/dataflatten"
+	"github.com/kolide/launcher/ee/tables/dataflattentable"
+	"github.com/kolide/launcher/ee/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+// I've set 3 different states for if the signature is verified:
+// VALID - The token parsed without errors and the signature was successfully validated.
+// INVALID - The signature attempted to validate with the matched public key, but it was a bad key.
+// UNKNOWN - The default state. This can mean that no key id matched, or simply no keys were provided to validate against.
+const (
+	Valid   = "VALID"
+	Invalid = "INVALID"
+	Unknown = "UNKNOWN"
+)
+
+// Values for include_raw_jwt column.
+var (
+	allowedIncludeValues = []string{"true", "false"}
+)
+
+// Created errors here to handle switching the verified value depending on the returned error.
+var (
+	ErrMissingKeyId     = errors.New("no key id found in the JWT header")
+	ErrMatchingKeyId    = errors.New("no key id matched the JWT header key id")
+	ErrParsingPemBlock  = errors.New("error parsing PEM block containing the public key")
+	ErrParsingPublicKey = errors.New("error parsing the public key from the PEM block")
+)
+
+type Table struct {
+	slogger *slog.Logger
+}
+
+func TablePlugin(slogger *slog.Logger) *table.Plugin {
+	columns := dataflattentable.Columns(
+		table.TextColumn("path"),
+		table.TextColumn("signing_keys"),
+		table.TextColumn("include_raw_jwt"),
```

Comment:
I added this in, but actually why would we want this as a column? I inserted it into the return row based on the `include_raw_jwt`. I think that makes it a lot cleaner as we can pivot the raw data instead of having it repeated in each row.

Reply:
Oh! Doing it pre-flatten so it's flattened into a row makes sense. Or possible manually. That's a good suggestion.